pomfiles/bcjmail-jdk15on.pom

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.bouncycastle</groupId>
+  <artifactId>bcjmail-jdk15on</artifactId>
+  <packaging>jar</packaging>
+  <name>Bouncy Castle Jakarta S/MIME API</name>
+  <version>${version}</version>
```

Comment:
The version variable will be replaced using sed
Normally, maven could have been used; but this project only supports ant based builds, so maven could not be used to set these variables